test/Project.toml

```diff
@@ -3,6 +3,7 @@ JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"
 JSONSchema = "7d188eb4-7ad8-530c-ae41-71a32a6d4692"
 Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
 NodeJS_18_jll = "c1e1d063-8311-5f52-a749-c7b05e91ae37"
+REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
```

Comment:
Having to do this isn't ideal. Were there no alternatives available to avoid it?

Reply:
So `REPL` is always loaded, since we use it in: https://github.com/PumasAI/QuartoNotebookRunner.jl/blob/a003fa8bbec452dfc79ba85c9fd279643198ee76/src/worker.jl#L12

That is the `Main` module where it's loaded. So the very straight-forward fix is to do `Main.REPL` wherever you have `REPL` currently rather than `import`ing it into the cell. See if that helps avoid adding the dep explicitly.